kahuna/public/js/components/gr-export-original-image/gr-export-original-image.js

```diff
@@ -0,0 +1,53 @@
+import angular from 'angular';
+
+import template from './gr-export-original-image.html!text';
+
+export const exportOriginalImage = angular.module('gr.exportOriginalImage', []);
+
+exportOriginalImage.controller('grExportOriginalImageCtrl', [
+    '$scope', '$rootScope', '$state', '$stateParams', 'mediaCropper',
+    function($scope, $rootScope, $state, $stateParams, mediaCropper) {
+        let ctrl = this;
+        var imageId = $stateParams.imageId;
```

Comment:
Why read the `$stateParams`, should the ID be read from the `image` attribute instead?